### PR TITLE
Changes to support Freebsd/PowerPC64

### PR DESCRIPTION
**Pull request by lenoil98**

### CODE REVIEW

File: src/core/sys/freebsd/execinfo.d [hunk 1]
```diff
@@ -43,7 +43,13 @@ else
             import ldc.llvmasm;
             __asm("str x29, $0", "=*m", &p);
         }
+        else version (PPC64) // LDC
+        {
+           import ldc.llvmasm;
+           __asm("stdu %r1,-144(%r1)", "=*m", &p);
```
Comment (kinke) [inline src/core/sys/freebsd/execinfo.d:49]: Can you explain this or link to a reference implementation? It looks like 'store stack pointer (%r1) into %r1-144' and as such an extremely brittle way to set `p`; `stdu %r1, $0` would probably make more sense in that case.

File: src/rt/sections_elf_shared.d [hunk 2]
```diff
@@ -1043,15 +1043,13 @@ else
...
-    }
-    else version (PPC64)
+
+version (linux) version (PPC) version = (linux_PPC_Any);
+version (linux) version (PPC64) version = (linux_PPC_Any);
+    
+    version (linux_PPC_Any)
     {
         extern(C) void* __tls_get_addr_opt(tls_index* ti) nothrow @nogc;
-        alias __tls_get_addr = __tls_get_addr_opt;
```
Comment (kinke) [inline src/rt/sections_elf_shared.d:1054]: This alias is still required. Also please indent the version declarations above (and remove the empty line before that block).

File: src/core/sys/freebsd/execinfo.d (hunk 1 above)
Comment (lenoil98) [reply src/core/sys/freebsd/execinfo.d:49]: I didn't create this code, I pulled from an earlier upstream update that was not committed.  I updated the code based on your comments.

Comment (kinke) [reply src/core/sys/freebsd/execinfo.d:49]: Okay, do you have a link? - And so you haven't tested this at all?

Comment (lenoil98) [reply src/core/sys/freebsd/execinfo.d:49]: See dialog in #3112 concerning the code mentioned above.  You may also want to confer with Redstar.

Yes, I've tested.  You can see comments in the PR #3112

Please also refer to https://github.com/dlang/druntime/pull/2683

**Pull request closed without merging**